src/nodes.coffee

```diff
@@ -2509,7 +2509,8 @@ exports.Op = class Op extends Base
     return @compileUnary        o if @isUnary()
     return @compileChain        o if isChain
     switch @operator
-      when '?'  then @compileExistence o
+      when '?'  then @compileExistence o      # Check if not null and not undefined
+      when '??' then @compileExistence o, yes # Check only if not undefined
```

Comment:
Hmm, I had thought this was an internal-only thing, like the grammar tokens like `yieldfrom`; I didn’t think I was creating a new operator in the language. Though I guess now that I inadvertently _have_ created a new operator, I guess we should discuss it 😄

I’m sure there’s some other way to pass down into `class Existence` that I want a ternary that only checks against `undefined`, not `undefined` or `null` (the legacy behavior). Or . . . should we keep this? On the one hand it definitely clutters the language to have two types of checking for existence, but on the other hand it _is_ the ES spec that sometimes things “exist” only when they’re not `undefined`, so maybe it would be useful to have this “ES existence” operator. What do you think?